### ISSUE TITLE
Get validation loss

### ISSUE THREAD

**Issue by nomercy89** [GeorgeSeif/Semantic-Segmentation-Suite#110]
Hello,

I need to get the validation loss, in this way i can measure overfitting.

So, at this code line:
`output_image = sess.run( network, feed_dict={net_input:input_image} )` 

How can i get the loss? Thanks
